README.md

```diff
@@ -57,3 +57,33 @@ Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_do
 and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
 be found at [https://hexdocs.pm/fetcher](https://hexdocs.pm/fetcher).
 
+
+## Run locally
+
+```sh
+# get dependencies
+mix deps.get
+
+# compile project
+mix compile
+
+# run the project
+mix run --no-halt
+
+# run the project interactive mode
+iex -S mix
+```
+
+## Local endpoints
+
+```sh
+curl localhost:8080/status
+
+curl -X POST --data '{"components":[{"id":"stream-icons","endpoint":"https://s3-eu-west-1.amazonaws.com/shared-application-buckets-public-1pmfwo80l61it/load-tests/static_envelopes/25082016/small-1.0.4.json","must_succeed":true}]}' localhost:8080/collect
```

Comment:
I've also added some test configs `test/fixtures/payload_multiple_small.json`
I _think_ you can use it in curl with:
```
curl -X POST --data @test/fixtures/payload_multiple_small.json`
```